orchagent/intfsorch.cpp

```diff
@@ -1219,11 +1220,47 @@ void IntfsOrch::removeRifFromFlexCounter(const string &id, const string &name)
     SWSS_LOG_DEBUG("Unregistered interface %s from Flex counter", name.c_str());
 }
 
+void IntfsOrch::cleanUpRifFromCounterDb(const string &id, const string &name)
+{
+    SWSS_LOG_ENTER();
+    string counter_key = getRifCounterTableKey(id);
+    string counter_backup_key = getRifCounterBackUpTableKey(id);
+    string rate_key = getRifRateTableKey(id);
+    string rate_init_key = getRifRateInitTableKey(id);
+    m_counter_db->del(counter_key);
+    m_counter_db->del(counter_backup_key);
+    m_counter_db->del(rate_key);
+    m_counter_db->del(rate_init_key);
+    SWSS_LOG_NOTICE("CleanUp interface %s oid %s from counter db", name.c_str(),id.c_str());
+}
+
 string IntfsOrch::getRifFlexCounterTableKey(string key)
 {
     return string(RIF_STAT_COUNTER_FLEX_COUNTER_GROUP) + ":" + key;
 }
 
+string IntfsOrch::getRifCounterTableKey(string key)
+{
+    return "COUNTERS:" + key;
+}
+
+string IntfsOrch::getRifCounterBackUpTableKey(string key)
```

Comment:
I don't see any references to COUNTERS_BACKUP table in the codebase. Can you please check?
https://github.com/Azure/sonic-swss/search?q=COUNTERS_BACKUP

Reply:
Thanks @dgsudharsan for the review.
As you pointed the COUNTERS_BACKUP table is not relevant, so removed its usage and submitted patch